### ISSUE TITLE
Limited neck extension

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5934]
**Preferred term label:** Limited neck extension

**Synonyms** Restricted neck extension; Neck extension limitation

**Definition (free text, please give PubMed ID)** Decreased ability to move the posterior of the head toward the back.

**Parent term (use hpo.jax.org/app)** Limitation of neck motion HP:0005986

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
